test_unstructured/partition/test_json.py

```diff
@@ -32,10 +35,23 @@
 is_in_docker = os.path.exists("/.dockerenv")
 
 
+# WARNING! You will need to set up API key, remove the cassettes, and re-run if
```

Comment:
Could you add the path to the cassettes so people know where to find them?

Reply:
Also, would it be helpful to have a small script that updates the cassettes and also scrubs out UNSTRUCTURED-API-KEY to make it less likely someone accidentally commits that?